hibike/unit_tests/hibike_message_tests.py

```diff
@@ -0,0 +1,386 @@
+"""
+Unit tests for functions in hibike_message.
+"""
+
+
+import unittest
+import random
+import queue
+import time
+import threading
+
+import serial
+
+import hibike_message
+import spawn_virtual_devices
+DEVICE_TYPES = list(hibike_message.DEVICES)
```

Comment:
I thought that `random.choice` only worked with lists, but upon closer inspection, it works with most sequences.